contracts/Controller.sol

```diff
@@ -310,7 +322,30 @@ contract Controller is ReentrancyGuard, Ownable {
      * @notice deposit long option into vault
      * @param _args DepositArgs structure
      */
-    // function _depositLong(Actions.DepositArgs memory _args) internal {}
+    function _depositLong(Actions.DepositArgs memory _args) internal returns (MarginAccount.Vault memory) {
+        require(_args.from == msg.sender, "Controller: depositor address and msg.sender address mismatch");
+
+        address whitelistModule = AddressBookInterface(addressBook).getWhitelist();
+        WhitelistInterface whitelist = WhitelistInterface(whitelistModule);
```

Comment:
You can probably have an internal function called `getWhitelist()` that returns a WhitelistInterface, also the same for MarginPoolInterface, I can see we will use this a lot in the Controller. 

Will only save you 1 line of code but I think it helps readability a bit,  the naming of `whitelistModule` and `whitelist` is not that helpful 😂

Reply:
Yeah I agree, I'm gonna refactor this to something more fancy than that in another PR 😋